### ISSUE TITLE
Icons not found

### ISSUE THREAD

**Issue by Flowneee** [ikrivosheev/pa-notify#4]
I noticed that notifications, sent by `pa-notify`, doesn't have any icons. I use `dunst`. I dug a little and noticed that all installed themes (Adwaita - default for dunst, Arc, elementary) - all have `*-symbolic` icons only (ie `audio-volume-low-symbolic`). I checked with another program (xfce4-power-manager) - it also send `-symbolic` icons. I was unable to understand how really should icons work, but it could be helpful to add some CLI option to use (basically append) `-symbolic` icons instead of usual (as far as i understood they are legacy now).

Also I think urgency could be configurable as well.

If all this makes sense to you, i'll submit PR

**Comment by ikrivosheev** [comment]
@Flowneee hello! This is strange... This is a function to choice icon. 

```C
static gchar*
notify_icon(gint volume)
{
    if (volume < 0) {
        return "audio-volume-muted";
    } else if (volume >= 66) {
        return "audio-volume-high";
    } else if (volume >= 33) {
        return "audio-volume-medium";
    } else {
        return "audio-volume-low";
    }
}
```

Do you have this icons?

**Comment by Flowneee** [comment]
That is the thing - no. These icons are in `/legacy` folder for some themes, which is not handled by `dunst`. Some doesn't have them at all. All "actual" icons are `*-symbolic`. As I understand these are mostly svg and can be resized, so they are somewhat a standard now.

Check list of files here https://archlinux.org/packages/extra/any/adwaita-icon-theme/

**Comment by ikrivosheev** [comment]
@Flowneee well, I think:

1. Replaced `icons` from `/legacy` to `*-symbolic` - is ok, without cli flag
2. `urgency could be configurable as well` - is ok, we can add cli options for this)

**Comment by Flowneee** [comment]
Cool, then I'll send another PR soon (it's ready, just not pushed to gh). Thanks!

**Comment by ikrivosheev** [comment]
> Cool, then I'll send another PR soon (it's ready, just not pushed to gh). Thanks!

I don't) Ok, I'm waiting on it)

**Comment by ikrivosheev** [comment]
@Flowneee can we close the issue?

**Comment by Flowneee** [comment]
I think so, yes.
